vendor/sunnyvalley/+POST_DEINSTALL.post

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Shouldn’t be needed with the shadow trick.

Reply:
when I make test using pkg remove, SunnyValley.conf could not delete.